katas/content/teleportation/reconstruct_message_phi_minus/solution.md

```diff
@@ -0,0 +1,10 @@
+To apply standard teleportation, entanglement pair should be brought back to $\ket{\Phi ^+}$. Correction with `Z` gate can help achieve the transformation from $\ket{\Phi ^-}$ to $\ket{\Phi ^+}$. 
```

Comment:
Let's rephrase a bit to emphasize the idea of "first bring back to Phi+, then use standard":
```suggestion
Bob's side of this protocol can actually be represented as a sequence of two steps:

 1. Transform the Bell pair he shares with Alice back to $\ket{\Phi ^+}$. This can be done by applying the $Z$ gate to Bob's qubit.
 2. Apply the standard teleportation protocol.

The final set of corrections we need is the $Z$ gate, followed by the standard teleportation corrections:
```